packages/api/lib/controllers/boxesController.js

```diff
@@ -184,9 +199,26 @@ const getBoxes = function getBoxes (req, res, next) {
   // content-type is always application/json for this route
   res.header('Content-Type', 'application/json; charset=utf-8');
 
+  // default format
+  let stringifier = jsonstringify({ open: '[', close: ']' });
+  // format
+  if (req._userParams.format === 'geojson') {
+    stringifier = jsonstringify({ open: '{"type":"FeatureCollection","features":[', close: ']}' }, geoJsonStringifyReplacer);
+  }
+
   Box.findBoxesLastMeasurements(req._userParams)
     .then(function (stream) {
+
+      if (req._userParams.classify === 'true') {
+        stream = stream
+          .pipe(classifyTransformer())
```

Comment:
Use `.pipe(new classifyTransformer())` to avoid calling of `instanceof` in classifyTransformer constructor